### PR TITLE
AWSPRIVATELINKENDPOINT

### PR DESCRIPTION
**Pull request by morecar**
### Checklist

* [x] I've read the guidelines and understand the acceptance criteria.
* [ ] The value of the attribute marked as `identifier` will be unique and valid. 
* [x] I've confirmed that my entity type wasn't already defined. If it is I'm providing an explanation above.

### CODE REVIEW

File: entity-types/infra-awsprivatelinkendpoint/golden_metrics.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+activeConnections:
+  title: Active Connections
+  unit: COUNT
+  queries:
+    aws:
+      select: latest(aws.privatelinkendpoints.ActiveConnections)
+      from: Metric
+      eventId: entity.guid
+      eventName: entity.name
+packetsDropped:
```
Comment (albertoolivan) [inline entity-types/infra-awsprivatelinkendpoint/golden_metrics.yml:55]: check those golde metric definitions, I see they are repeated

**Review by naxhh — CHANGES_REQUESTED** [review]
beep boop bop.

I have found some errors processing these changes:

 ```
* Error VALIDATING domainType: 'INFRA-RABBITMQQUEUE' on Golden metrics for key 'SELECT average(rabbitmq.message.delivered) FROM Metric WHERE entity.guid IN ('guid') FACET entity.name TIMESERIES' - Exception: The select clause uses a different value function than the rest.

* Error VALIDATING domainType: 'INFRA-RABBITMQQUEUE' on Golden metrics for key 'SELECT average(rabbitmq.message.published) FROM Metric WHERE entity.guid IN ('guid') FACET entity.name TIMESERIES' - Exception: The select clause uses a different value function than the rest.
```

File: entity-types/infra-awsprivatelinkendpoint/summary_metrics.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  unit: COUNT
+packetsDropped:
+  goldenMetric: packetsDropped
+  title: Packets Dropped
+  unit: COUNT
+rstPacketsReceived:
+  goldenMetric: rstPacketsReceived
+  title: Rst Packets Received
+  unit: COUNT
+bytesProcessed:
```
Comment (albertoolivan) [inline entity-types/infra-awsprivatelinkendpoint/summary_metrics.yml:43]: Here we have repeated summary metrics

**Review by naxhh — CHANGES_REQUESTED** [review]
beep boop bop.

I have found some errors processing these changes:

 ```
* Error VALIDATING domainType: 'INFRA-RABBITMQQUEUE' on Golden metrics for key 'SELECT average(rabbitmq.message.delivered) FROM Metric WHERE entity.guid IN ('guid') FACET entity.name TIMESERIES' - Exception: The select clause uses a different value function than the rest.

* Error VALIDATING domainType: 'INFRA-RABBITMQQUEUE' on Golden metrics for key 'SELECT average(rabbitmq.message.published) FROM Metric WHERE entity.guid IN ('guid') FACET entity.name TIMESERIES' - Exception: The select clause uses a different value function than the rest.
```

File: entity-types/infra-awsprivatelinkendpoint/summary_metrics.yml (hunk 2 above)
Comment (morecar) [reply entity-types/infra-awsprivatelinkendpoint/summary_metrics.yml]: Yeah, sorry, I was fixing the script that generates this files

**Review by albertoolivan — APPROVED** [review]

**Review by naxhh — APPROVED** [review]

**Review by naxhh — APPROVED** [review]

**Review by naxhh — APPROVED** [review]

**Review by naxhh — APPROVED** [review]

**Pull request merged**